<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMatchDialog</class>
 <widget class="QDialog" name="AddMatchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>834</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblHomeEmblem">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="lblAwayEmblem">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QToolBox" name="toolHome">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>274</width>
        <height>674</height>
       </rect>
      </property>
      <attribute name="label">
       <string>События</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="cmdAddHomeEvent">
         <property name="text">
          <string>Добавить событие</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lstHome"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>146</width>
        <height>133</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Состав</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="cmdHomeStart">
         <property name="text">
          <string>Закрепить состав</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lstHomeStart"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="locale">
      <locale language="Russian" country="RussianFederation"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lblHomeScore">
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="lblAwayScore">
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="lblAway">
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Jason</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblHome">
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Drakula</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6" colspan="3">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>274</width>
        <height>674</height>
       </rect>
      </property>
      <attribute name="label">
       <string>События</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="cmdAddAwayEvent">
         <property name="text">
          <string>Добавить событие</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lstAway"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>146</width>
        <height>133</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Состав</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="cmdAwayStart">
         <property name="text">
          <string>Закрепить состав</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lstAwayStart"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddMatchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddMatchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
